lib/pcsclite.js

```diff
@@ -12,7 +11,7 @@ var parse_readers_string = function(readers_str) {
     var pos;
     var readers = [];
     var ini = 0;
-    while ((pos = bt.indexOf(readers_str.slice(ini), '\0')) > 0) {
+    while ((pos = new Buffer(readers_str.slice(ini)).indexOf("\0")) > 0) {
```

Comment:
maybe use `Buffer.from()` here instead of new Buffer as there are plans to deprecate de constructor.